tools/site_admin/copy_pages.php

```diff
@@ -679,64 +679,15 @@ function do_stuff( $projectid_, $from_image_, $page_name_handling,
         }
     }
 
-    if ($merge_wordcheck_files) {
-        echo "<p>" . _("Merging wordcheck files.") . "</p>\n";
+    if ($merge_wordcheck_data) {
+        echo "<p>" . _("Merging WordCheck data.") . "</p>\n";
```

Comment:
Probably a good idea to be consistent and use "files and events" here, too. That gets all the squirrel-facing text consistent.